qiskit/providers/aer/noise/noise_model.py

```diff
@@ -19,6 +19,10 @@
 
 from qiskit.circuit import Instruction
 from qiskit.providers import BaseBackend
+try:
+    from qiskit.providers import Backend
```

Comment:
Could do something like:
```suggestion
try:
    from qiskit.providers import Backend as _Backend
    Backend = (BaseBackend, _Backend)
 except ImportError:
    Backend = BaseBackend

...
if isinstance(backend, Backend)
```

Reply:
Yeah that works too and would be a bit simpler, I'll update it now

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1170/commits/e92de0d92d826539fb4dafe3c194690872308c9f